topics/ada/index.md

```diff
@@ -0,0 +1,10 @@
+---
+aliases: ada-2012, ada2012, ada2005, ada95
```

Comment:
Doesn't look like `ada2005` is used on any public repositories. Can we remove it from the alias list?
```suggestion
aliases: ada-2012, ada2012, ada95
```